### ISSUE TITLE
Game specific configurations

### ISSUE THREAD

**Issue by PopeRigby** [benjamimgois/goverlay#61]
I don't know if this is already possible but it would be nice if GOverlay supported different configurations for each game.

**Comment by benjamimgois** [comment]
Not possible yet @PopeRigby , but i'll try to work on it in the future.

**Comment by Toadfield** [comment]
Yeah that would be really great,I hate to create separate config files for every game.

**Comment by yochananmarqos** [comment]
Might want to check out [steamtinkerlaunch](https://github.com/frostworx/steamtinkerlaunch#game-specific-use), it has support for that.

**Comment by Toadfield** [comment]
@yochananmarqos But that wouldn't work for non steam games.

**Comment by yochananmarqos** [comment]
@Toadfield Well, then [add them](https://support.steampowered.com/kb_article.php?ref=2219-YDJV-5557#:~:text=Launch%20Steam.,on%20%22Add%20Selected%20Programs%22.) to Steam. 😉

**Comment by Toadfield** [comment]
I don't want to add every game to steam,that would cost unecessary time
